README.md

```diff
@@ -0,0 +1,18 @@
+# bioutils
+
+A repository for generally useful bioinformatics tools
+
+# Description
+
+Here, we will consolidate scripts for commonly-used functions such as converting filetypes or calculating descriptive statitics.
+
+# Usage
+
+1. Go to https://github.com/maragkakislab/bioutils and fork the repository to your own Github account
+2. Clone the repository into `~/dev/` with `git clone https://github.com/[username]/bioutils`
```

Comment:
Better not to force the `~/dev` to the user. They can fork whenever they want. We can have `~/dev` as just a suggestion.